components/crm/.env.example

```
@@ -2,6 +2,16 @@ ENV_NAME=development
 # APP
 VERSION=v3.6.0
+# DEPLOYMENT MODE (controls TLS enforcement)
+# Valid values:
+#   - "saas": Lerian-hosted multi-tenant - TLS MANDATORY for all dependencies
+#             Service REFUSES TO START if any dependency lacks TLS
+#   - "byoc": Customer-hosted (Bring Your Own Cloud) - TLS recommended but not enforced
+#             Logs warnings for dependencies without TLS but allows startup
+#   - "local" (default): Developer workstation - TLS optional, no enforcement
+#
+# This setting controls the ValidateSaaSTLS() check at bootstrap.
+DEPLOYMENT_MODE=local
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEPLOYMENT_MODE key should go before the VERSION key

(UnorderedKey)